interactions/api/models/message.py

```diff
@@ -790,6 +790,19 @@ async def delete(
             guild_id=guild_id, emoji_id=int(self.id), reason=reason
         )
 
+    @property
+    def url(self) -> Optional[str]:
+        """
+        Returns the emoji's url.
+        :return URL of the emoji
+        :rtype: str
+        """
+        if not self._client:
+            raise AttributeError("HTTPClient not found!")
+        url = f"https://cdn.discordapp.com/emojis/{self.id}"
+        url += ".gif" if self.animated is True else ".png"
```

Comment:
```suggestion
        url += ".gif" if self.animated else ".png"
```

Reply:
Why removed ``is True`` though? ``self.animated`` returns boolean.

Reply:
It's one of Python's many magic!
`boolean = True`
`if boolean is True:` will return the same as `if boolean:`

Reply:
Ohhhhh...
Yeah, I don't expect this. Thanks for that sugggestion.